src/page-response-impact/ResponseImpactDashboard.tsx

```diff
@@ -86,6 +87,15 @@ const ResponseImpactDashboard: React.FC<Props> = ({
     setPopulationFormSubmitted(true);
   }
 
+  // current incarcerated population
+  const currentIncarceratedPop =
+    facilities &&
+    facilities.data &&
+    facilities.data.reduce((accumulator, facility) => {
```

Comment:
I _think_ with typescript you might be able to do `facilities?.data?.reduce()` and it would bail and return undefined if facilities.data is undefined.

Reply:
yes much better, thanks. I forget about that in TS sometimes.